### PR TITLE
[HOPSWORKS-1374] hudi examples

### PR DESCRIPTION
**Pull request by davitbzh**

### CODE REVIEW

File: notebooks/featurestore/hsfs/feature_engineering.ipynb [hunk 1]
```diff
@@ -310,28 +312,6 @@
...
-     "output_type": "stream",
-     "text": [
-      "<hsfs.feature_group.FeatureGroup object at 0x7fd3e4d40850>"
-     ]
-    }
-   ],
-   "source": [
-    "sales_part_fg_meta = fs.create_feature_group(name=\"sales_fg\",\n",
-    "                                        version=2,\n",
-    "                                        partition_key=['store'],\n",
```
Comment (moritzmeister) [inline notebooks/featurestore/hsfs/feature_engineering.ipynb:329]: Why did you remove the partition key example?

File: notebooks/featurestore/hsfs/time_travel_python.ipynb [hunk 2]
```diff
@@ -0,0 +1,1336 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Time travel operations in Hopsworks Feature Store\n",
+    "\n",
+    "In this notebook we will introduce time travel operations in Hopsworks Feature Store (HSFS). Currently HSFS supports Apache Hudi (http://hudi.apache.org/) a storage abstraction/library for doing **incremental** data ingestion to a Hopsworks Feature Store.\n",
+    "\n",
+    "TLDR; Hudi is a storage abstraction/library build on top of Spark. A Hudi dataset stores data in Parquet files and maintains additional metadata to make upserts efficient. A Hudi ingest job is intended to be run as a streaming ingest job, on an interval such as every 15 minutes, reading deltas from a message-bus like Kafka and ingesting the deltas **incrementally** into a data lake.\n",
+    "\n",
+    "![Incremental ETL](./../images/incr_load.png \"Incremetal ETL\")"
```
Comment (moritzmeister) [inline notebooks/featurestore/hsfs/time_travel_python.ipynb:13]: Can we get Lex to make a more modern version of these diagrams?

**Review by moritzmeister — APPROVED** [review]
Generally looks good, a bit much text on the Hudi notebooks in the beginning IMO.

In the next iteration we should replace the diagrams with more modern versions.

**Comment by SirOibaf** [comment]
On comment from my side. Can we please have a semi-standard code formatting for spark? I really find it hard to read if it doesn't have any sort of indentation.
I'd like to see this:
```
val classFg = (fs.createFeatureGroup()
                        .name("class_fg")
                        .description("Hudi Class Feature Group")
                        .version(1)
                        .primaryKeys(Seq("id"))
                        .partitionKeys(Seq("year"))
                        .timeTravelFormat(TimeTravelFormat.HUDI)
                        .build())
```
rather than this: 
```
val classFg = fs.createFeatureGroup().
name("class_fg").
description("Hudi Class Feature Group").
version(1).
primaryKeys(Seq("id")).
partitionKeys(Seq("year")).
timeTravelFormat(TimeTravelFormat.HUDI).
build();
```

File: notebooks/featurestore/hsfs/feature_engineering.ipynb (hunk 1 above)
Comment (davitbzh) [reply notebooks/featurestore/hsfs/feature_engineering.ipynb]: My bad. Its back now

File: notebooks/featurestore/hsfs/time_travel_python.ipynb (hunk 2 above)
Comment (davitbzh) [reply notebooks/featurestore/hsfs/time_travel_python.ipynb:13]: pinged Lex regarding this
